src/Middleware/NodeServices/ref/Microsoft.AspNetCore.NodeServices.netcoreapp3.0.cs

```diff
@@ -80,9 +93,12 @@ public abstract partial class OutOfProcessNodeInstance : Microsoft.AspNetCore.No
 }
 namespace Microsoft.Extensions.DependencyInjection
 {
+    [System.ObsoleteAttribute("Use Microsoft.AspNetCore.SpaServices.Extensions")]
     public static partial class NodeServicesServiceCollectionExtensions
     {
+        [System.ObsoleteAttribute("Use Microsoft.AspNetCore.SpaServices.Extensions")]
         public static void AddNodeServices(this Microsoft.Extensions.DependencyInjection.IServiceCollection serviceCollection) { }
+        [System.ObsoleteAttribute("Use Microsoft.AspNetCore.SpaServices.Extensions")]
         public static void AddNodeServices(this Microsoft.Extensions.DependencyInjection.IServiceCollection serviceCollection, System.Action<Microsoft.AspNetCore.NodeServices.NodeServicesOptions> setupAction) { }
```

Comment:
Same question as above - there is no analog in `Microsoft.AspNetCore.SpaServices.Extensions`. What should be used instead?